### PR TITLE
Save piece position

### PR DESCRIPTION
**Pull request by wookee55**
What I did:
- Added update_position method to the Piece model to update x_position and  y_position attribute

Question:
Currently, we're retrieve the piece by its x_position, y_position and game_id. Since the piece_id is being passed to the game controller in the ajax put request, can we use it to retrieve the piece instead?

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -12,56 +12,58 @@ def self.create_game(game_params)
...
-    Piece.create(type: 'Knight', x_position: 6, y_position: 7, user: black_player, game: self)
-    Piece.create(type: 'Bishop', x_position: 2, y_position: 0, user: white_player, game: self)
-    Piece.create(type: 'Bishop', x_position: 5, y_position: 0, user: white_player, game: self)
-    Piece.create(type: 'Bishop', x_position: 2, y_position: 7, user: black_player, game: self)
-    Piece.create(type: 'Bishop', x_position: 5, y_position: 7, user: black_player, game: self)
-    Piece.create(type: 'Queen', x_position: 3, y_position: 0, user: white_player, game: self)
-    Piece.create(type: 'Queen', x_position: 3, y_position: 7, user: black_player, game: self)
-    Piece.create(type: 'King', x_position: 4, y_position: 0, user: white_player, game: self)
-    Piece.create(type: 'King', x_position: 4, y_position: 7, user: black_player, game: self)
+    Piece.create(:type => 'Pawn',   :x_position => 0, :y_position => 1, user: white_player, game: self, is_white: true)
```
Comment (hoitomt) [inline app/models/game.rb]: Please don't change the hash syntax. Let's leave the semi-colons, no hash-rockets:

```
Piece.create(type: 'Pawn', x_position: 0, y_position: 1, user: white_player, game: self, is_white: true)
```

**Comment by wookee55** [comment]
Replaced hash rockets with colons. Is there an advantage to doing it one way over the other?

**Comment by joehenriod** [comment]
+ 1

**Comment by AUcod3r** [comment]
:+1:

**Pull request merged**